src/main/java/appeng/client/gui/implementations/GuiOreFilter.java

```diff
@@ -164,9 +165,19 @@ public void setTextFieldValue(final String displayName, final int mousex, final
         final int[] ores = OreDictionary.getOreIDs(stack);
 
         if (ores.length > 0) {
-            textField.setText(OreDictionary.getOreName(ores[0]));
+
+            if (Mouse.isButtonDown(0)) {
+                // left click
+                textField.setText(textField.getText() + OreDictionary.getOreName(ores[0]));
+            } else if (Mouse.isButtonDown(1)) {
+                // right click
+                textField.setText(OreDictionary.getOreName(ores[0]));
+            }
+
         } else {
             textField.setText(displayName);
```

Comment:
I recommend using similar behavior for the case when oredict is not found, so as not to break the user's expectations

Reply:
or ignore if item don't have oredict

Reply:
> I recommend using similar behavior for the case when oredict is not found, so as not to break the user's expectations

oh, i forget that😂

Reply:
> or ignore editing if item don't have oredict (because the item name in this input does not make sense)

this sounds more reasonable